src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -176,5 +177,36 @@ public static void OpenMachineRootStore_Permissions(OpenFlags permissions, bool
                 }
             }
         }
+
+        [Fact]
+        public static void MachineRootStore_NonEmpty()
+        {
+            // On everything except OS X we're using the normal system default trust store.
+            // So, if we've read things right, there should be a respectable amount of
+            // certificates in the store.
+            //
+            // On OSX this devolves into a "can we open the store and load the collection
+            // without throwing" test, but eventually we'll have tighter guarantees there.
+            //
+            // This test will fail on systems where the administrator has gone out of their
+            // way to prune the trusted CA list down below this threshold.
+            //
+            // As of 2016-01-25, Ubuntu 14.04 has 169, and CentOS 7.1 has 175, so that'd be
+            // quite a lot of pruning.
```

Comment:
I have 51 on my desktop, and 42 on my laptop.  It's a bit hard for me to really say what the default is, but it seemed well above 5. :smile: